app/controllers/requests/request_controller.rb

```diff
@@ -5,7 +5,8 @@
 
 module Requests
   class RequestController < ApplicationController
-    skip_before_action :verify_authenticity_token, only: [:borrow_direct]
+    skip_before_action :verify_authenticity_token, only: [:borrow_direct,:ill_to_edd]
```

Comment:
If we move away from a JSON response for to :ill_to_edd we should remove it from this filter.